app/app.scss

```diff
@@ -0,0 +1,163 @@
+@font-face {
```

Comment:
it's totally up to you, I don't intend to write css styles , that's not my part of the job, so use what you want and what will be best for your needs. Current code is only plain css copied into scss files ( in mostly cases ). So, back to main question, if you want postcss, let's use it ! :)